DEPRECATIONS.md

```diff
@@ -9,6 +9,7 @@ The following is a list of deprecations, according to the [Deprecation Policy](h
 | DEPPS3 | Config option `enforcePrivateUsers` defaults to `true` | [#7319](https://github.com/parse-community/parse-server/pull/7319)   | 5.0.0 (2022)                    | 6.0.0 (2023)                    | deprecated            | -     |
 | DEPPS4 | Remove convenience method for http request `Parse.Cloud.httpRequest` | [#7589](https://github.com/parse-community/parse-server/pull/7589)   | 5.0.0 (2022)                    | 6.0.0 (2023)                    | deprecated            | -     |
 | DEPPS5 | Config option `allowClientClassCreation` defaults to `false` | [#7925](https://github.com/parse-community/parse-server/pull/7925)   | 5.3.0 (2023)                    | 7.0.0 (2024)                    | deprecated            | -     |
```

Comment:
I just realized these dates are wrong, I think it should be (5.3.0) 2022 and 7.0.0 (2023), as version 6 is 2022, and 7 2023 cc @mtrezza

Reply:
Which dates? If unrelated to this PR, could you open a new PR (probably without a new issue and reference the original issue that added this line as reference)?

Reply:
Solved in #7992